### PR TITLE
Milestone 1 - Features

### PR DESCRIPTION
**Pull request by sergioobolevich**
Hello there!

In this pull request, I have created the React application base using NPM.
I have installed big.js and PropTypes libraries.
I have created an <App /> component that prints 'Hi' on the screen.
I have deployed the application to Heroku (live in here: https://react-calculator-sdog.herokuapp.com/) 

Best regards,
Sergio

### CODE REVIEW

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

__You've made a lot of progress so far :handshake: 

_**Top highlights:**_

- Descriptive PR  title and summary :100: 
- Correct usage of the `.gitignore` file :+1: 

__However, some changes are required before this project can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [x] Your project must have a descriptive Readme right from the first milestone of this project. Kindly add all necessary sections to your Readme file, including the live demo link.
- [x] You're having two(2) Readme files (`README.md`, and `README.old.md`). Kindly remove the duplicate.
- [x] You've not set up the linters configuration for this project. Kindly do so and ensure that all linter checks are passing successfully before you request another review.

**[ OPTIONAL CHANGES ]**

- [x] Against your subsequent commits, kindly ensure that your commit messages are well composed and descriptive. Personally, I believe that messages like `Initialize` do not look professional.

* * *
**Please make the required changes and submit another review request when you are done.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

File: tailwind.config.js
Comment (bellom) [inline tailwind.config.js:1]: You added this file for a reason, if I may ask, why?

It is weird to me to see this file because app initialized with create-react-app won't generate this.

Keep in mind that, this project required making use of the `create-react-app` script

File: craco.config.js
Comment (bellom) [inline craco.config.js]: Job well done!

You added this file for a reason, if I may ask, why?

It is weird to me to see this file because app initialized with create-react-app won't generate this.

File: package.json
Comment (bellom) [inline package.json]: Keep in mind that, this project required making use of the `create-react-app` script.

You might want to add it back, thanks.

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @chubaquelo   :clap: 

**Great work making `almost` all the necessary corrections from the previous review.**

# Status: Changes Required! ♻️ 

**Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

File: tailwind.config.js
Comment (sergioobolevich) [reply tailwind.config.js:1]: Hello Bellom, I added that because I want to use Tailwind for my UI. I followed Tailwind instructions here: https://tailwindcss.com/docs/guides/create-react-app 
The application was made with create-react-app.

File: craco.config.js
Comment (sergioobolevich) [reply craco.config.js]: Hello Bellom, I added that because I want to use Tailwind for my UI. I followed Tailwind instructions here: https://tailwindcss.com/docs/guides/create-react-app 
The application was made with create-react-app.

File: package.json
Comment (sergioobolevich) [reply package.json]: Hello Bellom, I added that because I want to use Tailwind for my UI. I followed Tailwind instructions here: https://tailwindcss.com/docs/guides/create-react-app 
The application was made with create-react-app.

**Review by arslanbisharat — APPROVED** [review]
# Approved :clap:

Hello Developer, :wave:

Thank You for submitting this Pull Request. I am thrilled to announce that your project has met all the minimum requirements and now it is approved. :clap:

I am wishing you very good luck with the next project. :smile:

Once again, Congratulations!:joy:

**Best,
Arslan Bisharat(PT-TSE)
Slack: @arslanbisharat**

**Pull request merged**
